org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,136 @@
+{
+    "app-id": "org.sparkleshare.Sparkleshare",
+    "branch": "master",
+    "command": "sparkleshare",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "metadata": "metadata",
+    "separate-locales": false,
```

Comment:
Does this application not install translations into the standard locations?

Reply:
No translations at all at this point.

Reply:
Then you can just remove this.